From 930e9fae6babcffc6b74823d45d3bbf394e05cc9 Mon Sep 17 00:00:00 2001
From: Thomas Woerner <twoerner@redhat.com>
Date: Mon, 22 May 2017 18:05:38 +0200
Subject: [PATCH 4/6] firewalld.dbus: Add missing properties
 nf_conntrach_helper_setting and nf_conntrack_helpers

(cherry picked from commit 89a186db02dd3776dce4105d1266b4863b3b4e8b)
---
 doc/xml/firewalld.dbus.xml | 10 +++++++++-
 1 file changed, 9 insertions(+), 1 deletion(-)

diff --git a/doc/xml/firewalld.dbus.xml b/doc/xml/firewalld.dbus.xml
index de18ab2d514a..52b5b3b0f955 100644
--- a/doc/xml/firewalld.dbus.xml
+++ b/doc/xml/firewalld.dbus.xml
@@ -459,7 +459,15 @@
             <term><parameter>IPv6ICMPTypes</parameter> - as - (ro)</term>
             <listitem><para>The list of supported IPv6 ICMP types.</para></listitem>
           </varlistentry>
-          <varlistentry id="FirewallD1.Properties.interface_version">
+	  <varlistentry id="FirewallD1.Properties.nf_conntrach_helper_setting">
+            <term><parameter>nf_conntrach_helper_setting</parameter> - b - (ro)</term>
+            <listitem><para>Kernel nf_conntrack_helper setting.</para></listitem>
+          </varlistentry>
+          <varlistentry id="FirewallD1.Properties.nf_conntrack_helpers">
+            <term><parameter>nf_conntrack_helpers</parameter> - a{sas} - (ro)</term>
+            <listitem><para>The list of conntrack helpers supported by the kernel.</para></listitem>
+          </varlistentry>
+	  <varlistentry id="FirewallD1.Properties.interface_version">
             <term><parameter>interface_version</parameter> - s - (ro)</term>
             <listitem><para>firewalld D-Bus interface version string.</para></listitem>
           </varlistentry>
-- 
2.12.0

